### PR TITLE
Leads login

### PR DESCRIPTION
**Pull request by devikasharma2890**

### CODE REVIEW

File: src/app/features/login-form/login-form.component.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+export const users: User[] = [
+        {userName: 'admin@admin.com', password: 'admin123', firstName: 'Devika', lastName: 'Sharma', profileImage: ''},
+        {userName: 'admin2@admin.com', password: 'admin345', firstName: 'Ridhi', lastName: 'Sharma', profileImage: ''}
+    ];
+
+@Component({
+  selector: 'login-form',
+  templateUrl: './login-form2.component.html',
+  providers: [AuthenticationService],
```
Comment (rakeshaggarwal1980) [inline src/app/features/login-form/login-form.component.ts]: name of html file and ts files should have same initials. Currently html file is having prefix login-form2 and ts file is having prefix login-form

Comment (devikasharma2890) [inline src/app/features/login-form/login-form.component.ts]: done

File: src/app/services/loginAuth.service.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import {Router} from '@angular/router';
+import { User } from '../classes/user.model';
+
+@Injectable()
+export class AuthenticationService {
+        constructor(private _router: Router) {}
+        logout() {
+            localStorage.removeItem('loggedinUser');
+            this._router.navigate(['login-form']);
+        }
```
Comment (rakeshaggarwal1980) [inline src/app/services/loginAuth.service.ts]: Navigating from one screen to another screen should be the responsibility of current screen, not service. Service should have functionality for checking if user is valid or not and at most it should store the profile information init that can be used later on

File: src/app/services/loginAuth.service.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            if (authenticatedUser && authenticatedUser.password === user.password) {
+                localStorage.setItem('loggedinUser',authenticatedUser.userName);
+                console.log('User is authenticated');
+                this._router.navigate(['user-dashboard']);
+                return true;
+            }
+            return false;
+        }
+        checkCredentials() {
+            if (localStorage.getItem('loggedinUser') === null) {
```
Comment (rakeshaggarwal1980) [inline src/app/services/loginAuth.service.ts]: we don't need this method at all

File: src/app/services/loginAuth.service.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+@Injectable()
+export class AuthenticationService {
+        constructor(private _router: Router) {}
+        logout() {
+            localStorage.removeItem('loggedinUser');
+            this._router.navigate(['login-form']);
+        }
+        login(user,users) {
+            var authenticatedUser = users.find(eachUser => eachUser.userName === user.userName);
```
Comment (rakeshaggarwal1980) [inline src/app/services/loginAuth.service.ts]: we should not pass collection of users to login method...service should know from where to pick list of users. Only details of user to be logged in should be passed

**Pull request closed without merging**